### PR TITLE
Added GraphStore Function

### PR DESCRIPTION
**Pull request by wangxiaoyunNV**
I added a neighbor_sampler function to the graph store class.

### CODE REVIEW

**Comment by BradReesWork** [comment]
rerun tests

**Review by BradReesWork — CHANGES_REQUESTED** [review]
need more expansive test cases
* test ndata
* test edata
* test sampling

**Comment by BradReesWork** [comment]
rerun tests

File: python/cugraph/cugraph/gnn/graph_store.py [hunk 1]
```diff
@@ -15,6 +15,9 @@
 import cugraph
 from cugraph.experimental import EXPERIMENTAL__PropertyGraph as PropertyGraph
```
Comment (BradReesWork) [inline python/cugraph/cugraph/gnn/graph_store.py:16]: you can do  from _cugraph.experimental import PropertyGraph_

Comment (wangxiaoyunNV) [reply python/cugraph/cugraph/gnn/graph_store.py]: I just changed it.

**Comment by BradReesWork** [comment]
rerun tests

**Review by BradReesWork — APPROVED** [review]

**Comment by BradReesWork** [comment]
@gpucibot merge

**Pull request merged**